userfiles/minc_file/views/_volume_viewer.html.erb

```diff
@@ -461,20 +473,20 @@
 
       if (BrainBrowser.utils.isFunction(volume.getWorldCoords)) {
         world_coords = volume.getWorldCoords();
-        $("#world-x-" + vol_id).val(world_coords.x.toPrecision(6));
-        $("#world-y-" + vol_id).val(world_coords.y.toPrecision(6));
-        $("#world-z-" + vol_id).val(world_coords.z.toPrecision(6));
+        $("#world-x-<%= userfile_id %>").val(world_coords.x.toPrecision(6));
```

Comment:
If vol_id was replaced everywhere (is it?), do we still need the line

```javascript
var vol_id = panel.volume_id;
```

above, near the beginning of the function?